deploy/crds/org.eclipse.che_v1_checlusterbackup_cr.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: org.eclipse.che/v1
+kind: CheClusterBackup
+metadata:
+  name: eclipse-che-backup
+spec:
+  # If true will setup REST backup server pod and add configuration to use it
+  autoconfigureRestBackupServer: false
+  # Set to true to start backup process
+  triggerNow: false
+  # Might be omitted if only one server configured
+  serverType: 'sftp'
+  # Usually only one server is needed
```

Comment:
Example is incorrect. Since it must contain only one backup configuration

Reply:
Maybe we should keep all servers configured as an example and just warn user in comment that only one configuration at a time allowed?

Reply:
ok